### PR TITLE
ddl: store  default value as string when value type is binary or bit

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
In v2.1.4, execute below SQL will be panic: v2.1.5 and after won't panic.
```sql
create table t (a int default b'0');
show columns from t;    # panic here
```

The main reason of panic is we think the ColumnInfo.DefaultValue is a string type( you can check the code of ColumnInfo.DefaultValue usage place ). but in upper SQL situation, the default value will be uint64, and after json Marshal and Unmarshal, the default value type will be float64.  That's not what would expect.

Below was a simple example of json marshal and unmarshal:

```go
package main

import (
	"encoding/json"
	"fmt"
	"reflect"
)

type Column struct {
	Default interface{} `json:"default"`
}

func main() {
	c := Column{
		Default: uint64(1),
	}
	data, _ := json.Marshal(c)
	c1 := Column{}
	json.Unmarshal(data, &c1)
	fmt.Printf("%v\n", reflect.TypeOf(c1.Default))    // the output is float64.
}
```

### What is changed and how it works?
Change `getDefaultValue` function to convert uint64 to string before return.

### Check List 

Tests 

 - Unit test

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: ddl/ddl_api.go [hunk 1]
```diff
@@ -533,8 +534,12 @@ func getDefaultValue(ctx sessionctx.Context, colName string, c *ast.ColumnOption
 			// its raw string content here.
 			return v.GetBinaryLiteral().ToString(), nil
 		}
-		// For other kind of fields (e.g. INT), we supply its integer value so that it acts as integers.
-		return v.GetBinaryLiteral().ToInt(ctx.GetSessionVars().StmtCtx)
+		// For other kind of fields (e.g. INT), we supply its integer as string value.
+		value, err := v.GetBinaryLiteral().ToInt(ctx.GetSessionVars().StmtCtx)
```
Comment (XuHuaiyu) [inline ddl/ddl_api.go]: It seems that the default value is `TypeBit`, why won't it go through line 528?

Comment (crazycs520) [reply ddl/ddl_api.go]: The value type is `typeBit`, but the column type is `Int`. 
eg: `create table t (a int default b'1');`

**Pull request merged**